utils.go

```diff
@@ -29,3 +30,14 @@ func removeBranchHash(gitURL string) (out string) {
 	out = strings.Split(out, "@")[0]
 	return
 }
+
+func getGoPath() (string, error) {
+	goEnvCmd := exec.Command("go", "env", "GOPATH")
```

Comment:
We should move this logic to our filewrapper lib, and also move that out of gomu and into our hatchify org

Reply:
Ill move filewrapper to repoMan, and we can do all our git and go.mod stuff from repoMan :)